lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -507,6 +511,23 @@ data TransactionInfo = TransactionInfo
     -- ^ Creation time of the block including this transaction.
     } deriving (Show, Eq, Ord)
 
+data SortDirection = Ascending | Descending
+    deriving (Eq, Show)
```

Comment:
We are "hard-coding" `Descending` in quite a few places for being the default direction. What about having an explicit top-level identifier `defaultDirection = Descending` and use that `defaultDirection` everywhere else the semantic applies?

Reply:
Great idea I think!